templates/seL4SharedDataWithCaps.template.c

```diff
@@ -52,12 +52,12 @@
 /*? macros.shared_buffer_symbol(sym=dataport_symbol_name, shmem_size=shmem_symbol_size, page_size=page_size) ?*/
 
 /*- set arch_name = obj_space.spec.arch.capdl_name() -*/
-/*- if arch_name in ['ia32', 'x86_64'] -*/
+/*- if arch_name in ['aarch32', 'aarch64', 'arm_hyp', 'ia32', 'x86_64'] -*/
     /*- set cached_default = true -*/
 /*- else -*/
     /*- set cached_default = false -*/
 /*- endif -*/
-/*- set cached = configuration[me.instance.name].get('%s_hardware_cached' % me.instance.name, cached_default) -*/
+/*- set cached = configuration[me.instance.name].get('%s_hardware_cached' % me.interface.name, cached_default) -*/
```

Comment:
Should this be a separate commit, as it seems to fix a bug?

Reply:
Did you mean separate PR, it is a separate commit already? Waiting for comments also from @kent-mcleod. Actually the other one, making the mapping cached, is kind of bug fix also, because I am quite confident vm-introspect example from camkes-vm-examples would not have been running OK on ARM. Might have worked for most of time, but has potential to misbehave, as is common when you mix cached and non-cached mappings.

Reply:
It's just the change from `instance` to `interface` here that I was wondering about. Seem unrelated to any caching.

Reply:
Ah no. just ignore my comment. Seem I was looking at the wrong commit.